### PR TITLE
env: Add MustGet functions to env package

### PR DESCRIPTION
**Pull request by ryanslade**
Used in gitserver as an example.

### CODE REVIEW

File: internal/env/env.go [hunk 1]
```diff
@@ -128,6 +129,26 @@ func Get(name, defaultValue, description string) string {
...
+	}
+	return d
+}
+
+// MustGetInt is similar to Get but ensures that the value is a valid int.
+func MustGetInt(name string, defaultValue int, description string) int {
+	s := Get(name, strconv.Itoa(defaultValue), description)
+	i, err := strconv.Atoi(s)
+	if err != nil {
+		panic(fmt.Sprintf("parsing int from environment %q, %q", name, s))
```
Comment (arussellsaw) [inline internal/env/env.go:147]: this message could be a bit more user friendly if we expect this to crash the app for users setting up the first time

Comment (ryanslade) [reply internal/env/env.go:147]: Maybe:

```go
panic(fmt.Sprintf("unable to parse int from %q, value %q", name, s))
```

What do you think?

Comment (arussellsaw) [reply internal/env/env.go:147]: might be worth including the description of the option as well

Comment (ryanslade) [reply internal/env/env.go]: I think the description might be too much text. What do you think of the new version I just pushed?

Comment (arussellsaw) [reply internal/env/env.go]: fair enough, yeah i think that's a decent bit more understandable 👍

**Pull request merged**